plugins/inputs/opcua/sample.conf

```
@@ -85,3 +85,6 @@
   # [inputs.opcua.workarounds]
     ## Set additional valid status codes, StatusOK (0x0) is always considered valid
     # additional_valid_status_codes = ["0xC0"]
+
+    ## Use regular reads instead of registered reads
+    # use_regular_reads = true
```

The example conf should represent the default values for commented out parameters:
```suggestion
    # use_regular_reads = false

```